doc/cm/cm_wifi.yml

```diff
@@ -0,0 +1,143 @@
+---
+# SPDX-License-Identifier: Apache-2.0
+
+- comment: |
+    WiFi AP and STA settings on Agents.
+
+    Copyright (C) 2025 Interpretica, Unipessoal Lda. All rights reserved.
+
+
+- register:
+
+    - oid: "/agent/wifi"
+      access: read_only
+      type: none
+      d: |
+         Root object of the WiFi configuration tree.
+         Name: empty
+
+    - oid: "/agent/wifi/port"
+      access: read_create
+      type: none
+      d: |
+         WiFi port instance.
+         Name: any, typically 2_4 or 5
+
+    - oid: "/agent/wifi/port/enable"
+      access: read_write
+      type: string
+      d: |
+         Enable or disable the port
+         Name: empty
+         Value: 0 or 1
+
+    - oid: "/agent/wifi/port/ifname"
```

Comment:
This should be related to `/agent/interface` in some way/

Reply:
@ol-arteman in general it is not guaranteed that the port would always be mapped to the existing interface. Depending on configurator, it may run it later, making it impossible to bind to.